src/Kit/System/Watchdog/Supervisor.h

```diff
@@ -0,0 +1,125 @@
+#ifndef KIT_SYSTEM_WATCHDOG_SUPERVISOR_H_
+#define KIT_SYSTEM_WATCHDOG_SUPERVISOR_H_
+/*------------------------------------------------------------------------------
+ * Copyright Integer Fox Authors
+ *
+ * Distributed under the BSD 3 Clause License. See the license agreement at:
+ * https://github.com/Integerfox/kit.core/blob/main/LICENSE
+ *
+ * Redistributions of the source code must retain the above copyright notice.
+ *----------------------------------------------------------------------------*/
+/** @file */
+
+#include "kit_config.h"
+#include "Kit/System/Watchdog/WatchedThread.h"
+#include "Kit/Container/SList.h"
+#include "Kit/System/Mutex.h"
+#include <cstdint>
+
+/** Tick divider to reduce monitoring frequency. In this context a 'tick' means
+    that the 'Supervisor' thread's executes a single iteration if its event loop.
+    This is not the same thing as the platform system tick or any fixed interval
+    of time.  See the Kit::System::EventLoop for more details on how an Event
+    Loop executes
+ */
+#ifndef OPTION_KIT_SYSTEM_WATCHDOG_SUPERVISOR_TICK_DIVIDER
+#define OPTION_KIT_SYSTEM_WATCHDOG_SUPERVISOR_TICK_DIVIDER 10
+#endif
+
+///
+namespace Kit {
+///
+namespace System {
```

Comment:
Namespaces map 1-to-1 directory names.  Since this class is in the `src/Kit/System/Watchdog/` directory -->its namespace needs to be `Kit::System::Watchdog`.  Apply this change to all files in the this directory
```suggestion
///
namespace Kit {
///
namespace System {
///
namespace Watchdog {
```